_redirects

```diff
@@ -62,3 +62,4 @@
 /docs/future/server-bundles           /docs/guides/server-bundles          302
 /docs/future/spa-mode                 /docs/guides/spa-mode                302
 /docs/future/vite                     /docs/guides/vite                    302
+/docs/guides/fog-of-war               /docs/guides/lazy-route-discovery    302
```

Comment:
Given that this was always a 404 I don't think we need to setup a custom redirect for it

```suggestion

```